src/concepts/nodes.md

```diff
@@ -0,0 +1,72 @@
+# Nodes
+
+## What are Nodes?
+
+---
+
+**Nodes** are simply **actions** or **conditions**. By connecting Nodes together complex action sequences based on certain conditions are created.
+
+Nodes are visually represented by Caido as draggable blocks, colorized by category. They utilize an input/output model that can be used to send data altered by one Node to a subsequent Node.
+
+<img alt="Convert workflow" src="/_images/workflow_convert_basic.png" center/>
+
+## Connecting Nodes
+
+---
+
+A `Connection` is visually represented by the gray line between nodes and determine the order of execution.
+
+- Workflows use a top-down heirachical structure (_the Node at the very top represents the beginning of the flow and the Node at the bottom represents the end of the flow_).
```

Comment:
I am not a super fan of those single indented dots, but I am not sure what to use instead. Maybe we don't need anything?

Reply:
I think for the structure there should be something to make it easier on the eyes to parse through. Also helps to delineate subsets of information pertaining to a parent. Maybe just a differently shaped bullet point?

Reply:
Ah wait, misread. Yeah I agree. Will remove it.